### PR TITLE
Update help text for NI Binary Git Diff

### PR DESCRIPTION
**Pull request by shastriUF**
The current help text says to contact Humberto. We now have instructions in the wiki, so I'm updating the help text to point to the wiki (without including the internal-only link). I tested that searching for that string in the NI Azure DevOps wiki will return the instructions we need as the first result.

### CODE REVIEW

**Review by humberto-garza — APPROVED** [review]

File: src/azdo-pr-dashboard.user.js [hunk 1]
```diff
@@ -552,7 +552,7 @@
         swal.fire({
           title: 'This is a preview feature!',
           icon: 'warning',
-          text: 'You need to install the "NIBinary.GitDiff.reg" Protocol Handler first. Please talk to Humberto Garza to get it.',
+          text: 'You need to install the "NIBinary.GitDiff.reg" Protocol Handler first. To get it, please talk to Humberto Garza, or search for "Launch NI Binary Git Diff" in the Azure DevOps NI Wiki for instructions.',
```
Comment (alejandro5042) [inline src/azdo-pr-dashboard.user.js:555]: Swal (our dialog library) has an option for an HTML dialog. I suggest we add a `<a target="_blank" href="LINK_TO_WIKI">TEXT</a>` link. A link is more useful than a search term; and our links are now permalinks (with the ID in the wiki page).

Tip: Change `text:` to `html:`. For more docs: https://sweetalert2.github.io/#configuration

**Review by alejandro5042 — CHANGES_REQUESTED** [review]
Thank you for this change 👍 

Please follow the steps for a PR as posted by the welcome bot above: https://github.com/alejandro5042/azdo-userscripts/blob/master/CONTRIBUTING.md#testing-before-a-pr

Specifically, you need to bump the patch version in the userscript.

File: src/azdo-pr-dashboard.user.js (hunk 1 above)
Comment (alejandro5042) [reply src/azdo-pr-dashboard.user.js:555]: Please test this too 😄 I'm assuming it works the way it says it does.

Comment (humberto-garza) [reply src/azdo-pr-dashboard.user.js:555]: Are we okay with sharing the actual permalink here?

Comment (alejandro5042) [reply src/azdo-pr-dashboard.user.js:555]: I'm OK with it. We already share a _little_ bit about our AzDO instance (e.g. it is at `ni.visualstudio.com`), our process (e.g. `niconfig` and intermittent failure file paths) and besides _everything_ is behind authentication. The links don't reveal any IP. Besides, the permalink can be stripped to only contain the ID number and not even the page title:

e.g. https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/16924

**Review by alejandro5042 — APPROVED** [review]

**Pull request merged**